docs/source/documentation/concepts.rst

```diff
@@ -177,3 +177,39 @@ There are two ways to run several tasks in parallel on a same organization. The
 .. TODO:: Detail vertical and horizontal scaling
 
 .. TODO:: Explain what is substra tools
+
+
+Compute plan execution
+-----------------------
+
+Read this paragraph to understand what happens during the compute plan execution and what can be done to improve the execution time.
+
+Once a compute plan is submitted to the platform, its tasks are scheduled to be executed on each organization.
+
+On each organization, Substra fetches the assets needed for the first task, builds the Docker image of the algo and creates a container with the relevant assets. The task executes and Substra saves its outputs.
+Afterwards, every task **from the same compute plan** that uses the same algo is executed in the same Docker container.
+
+Asset preparation
+^^^^^^^^^^^^^^^^^^
+
+The first step of the task execution is to fetch the necessary assets.
+These include the inputs (e.g. the algo or opener files), the output of other tasks (input artifacts of the task) and data samples.
```

Comment:
I don't know if it's just the GitHub rendering but the e.g. part parenthesis and dots are not visible.

Reply:
they render well on the doc page: https://owkin-substra-documentation--173.com.readthedocs.build/en/173/documentation/concepts.html#compute-plan-execution